packages/components/src/hooks/useSecondsCountDown/useSecondsCountDown.ts

```diff
@@ -0,0 +1,49 @@
+import { useEffect, useState } from 'react';
+
+import { SecondsCountDown, type SecondsCountDownParams } from '../../utils';
+
+type UseSecondsTimerOptions = Pick<
```

Comment:
```suggestion
type UseSecondsTimerParams = Pick<
```

https://kaluga-astral.github.io/style-guide/docs/rules/react/hooks#%D1%82%D0%B8%D0%BF-%D0%B2%D1%85%D0%BE%D0%B4%D0%BD%D1%8B%D1%85-%D0%BF%D0%B0%D1%80%D0%B0%D0%BC%D0%B5%D1%82%D1%80%D0%BE%D0%B2-%D1%85%D1%83%D0%BA%D0%B0-%D0%B4%D0%BE%D0%BB%D0%B6%D0%B5%D0%BD-%D0%B8%D0%BC%D0%B5%D1%82%D1%8C-%D0%BF%D0%BE%D1%81%D1%82%D1%84%D0%B8%D0%BA%D1%81-params